Applications/CLI/Tests.cmake

```diff
@@ -126,7 +126,29 @@ if(NOT OGS_USE_MPI)
         )
         endforeach()
 
+    # Some Neumann BC tests
+    AddTest(
+        NAME NeumannBC_cube_top
+        PATH Elliptic/cube_1x1x1_NeumannBC
```

Comment:
The test is a GroundWaterFlow test and should be placed in Tests/Data/Elliptic/cube_1x1x1_GroundWaterFlow.